src/script/standard.cpp

```diff
@@ -312,10 +336,11 @@ CScript GetScriptForMultisig(int nRequired, const std::vector<CPubKey>& keys)
 {
     CScript script;
 
-    script << CScript::EncodeOP_N(nRequired);
+    script << nRequired;
```

Comment:
Why does this change (and L342) belong in this PR?

Reply:
Look at the implementation of `operator<<` in `CScript`. It handles all integers while `EncodeOP_N` only supports small (<=16) ones.